viewer/viewer.js

```diff
@@ -8068,6 +8068,45 @@ function updateHuntStatus (req, res, status, successText, errorText) {
   });
 }
 
+function validateUserIds (userIdList) {
+  return new Promise((resolve, reject) => {
+    const query = {
+      _source: ['userId'],
+      from: 0,
+      size: 10000,
+      query: { // exclude the shared user from results
+        bool: { must_not: { term: { userId: '_moloch_shared' } } }
+      }
+    };
+
+    Db.searchUsers(query, (error, users) => {
```

Comment:
maybe handle somewhere below, but does this handle the case when there are no users because they run anonymously? or is an anonymous user created anyway right away.